saleor/graphql/account/mutations/account/account_update.py

```diff
@@ -91,3 +94,22 @@ def perform_mutation(cls, root, info: ResolveInfo, /, **data):
         user = cast(models.User, user)
         data["id"] = graphene.Node.to_global_id("User", user.id)
         return super().perform_mutation(root, info, **data)
+
+    @classmethod
+    @traced_atomic_transaction()
+    def save(cls, info: ResolveInfo, instance: models.User, cleaned_input):
+        manager = get_plugin_manager_promise(info.context).get()
+
+        default_addresses = cls.save_and_apply_addresses_from_input(
+            cleaned_input=cleaned_input, user_instance=instance
+        )
+
+        if default_addresses["billing"]:
+            instance.addresses.add(default_addresses["billing"])
+        if default_addresses["shipping"]:
+            instance.addresses.add(default_addresses["shipping"])
+
+        instance.search_document = prepare_user_search_document_value(instance)
+        instance.save()
```

Comment:
Could we save only updated fields instead of the whole instance? If it will extend the scope too much, we can create a ticket for that